core/src/test/resources/compare/model-sc1-new.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+# Test to check default provider attribute changed
+topology:
+  SecurityConfiguration:
+    Realm:
+      myrealm:
+        # Version 1.9 should be used fixing NPE
+        AuthenticationProvider:
+          DefaultAuthenticator:
+            DefaultAuthenticator:
+              ControlFlag: REQUIRED
```

Comment:
how about some comments to clarify the test, like
```
# Changing the value of ControlFlag should cause this entire
# AuthenticationProvider section to be included in the compare result.
```
(please correct if any of these are inaccurate)